meetings/2021-11-03.md

```diff
@@ -0,0 +1,204 @@
+
+* KK: That's the client side? Right.. okay.
+
+* SC: General agreement to support N3 patch.
+
+* RV: Do I need to contribute to spec text?
+
+* KK: Yes. That'd be good.
+
+ACTION: RV to make initial draft. KK to assign relevant issues to RV. KK to record 'something' on that issue.
+
+
+### Write methods as requirements
+URL: https://github.com/solid/specification/pull/304
+
+* KK: Also for testing for compliance... if we have to test for eventuality that Solid server that does not support write operations then that makes the test suite more complex. We would want to support servers that are read-only but that is mostly just a Web server right.. At some point we want to introduce conformance classes. In addition, we want to have language that allows servers for read-only. Which is also in the spec.. Therefore to not tackle more than this then I'd like write methods for required.
+
+* RV: That clarifies a lot. I can imagine read-only servers. How do we know if a solid server announces whether htey are read or write?
+
+* KK: accept headers.
+
+* RV: or status codes.
+
+* SC: Test suite shouldn't be driving factor for the requirement
+
+* KK: I disagree with that idea because. The test suite is important for the health of the ecosystem. As a principle that is important but if there is no easy way to automate the testing then it is a toy.. you don't have a production system you have a toy.
+
+* SC: spec comes before test.. even has test modes have different ways of checking.
+
+* KK: there is principle and pragmatism. In general I agree to adopt the test to requirements. But if we go to the 
+
+* SC: then we are requiring automated test - that needs to be a specific requirement of what we expect from the requirement.
+
+* KK: what can we do that's testable in one week. instead of later on. if we want to do thta. so it does infuence a lot of stuff. I have spoken.
+
+* JB: Trying think of a case where
+
+* DZ: test sensor..
+
+* SC: some mentioned in the PR/issues.
+
+* JB: if you can 405 and not .. would you set out a case where you have a solid server and just have only read-only. or is it authz to make it read-only.
+
+* KK: I suspect that you can configure apache with configuration only to be read-only. yes, that thing will exist. at some point we just need a conformance class to have a read-only then that is very
+
+* SC: ...
+
+* RV: use case is read-only / private.
+
+* KK: that makes it more complex to implement.
+
+* DZ: 405?
+
+* SC: yes
+
+* RV: but maybe it needs to be 403 not 405 if we say that write is required.
+
+* JB: okay with 405 and not against 403.
+
+* RV: fine with it.
+
+* KK: language in my patch leaves it open so that can be defined in different terms.
+
+* DZ: i'm on the side of 405 because 403 notes to me with diff credentials it oculd work.
+
+* SC: 403 can be forbidden for any reason. not just authorization.
+
+* JB: in our case 403 seems more associated with authz butnot strictly either.
```

Comment:
```suggestion
* DZ: i'm on the side of 405 because 403 notes to me with diff credentials it oculd work.

* SC: `403` can be forbidden for any reason, not just authorization.

* JB: in our case, `403` seems more associated with authz but not strictly either.
```